cdci_data_analysis/analysis/parameters.py

```diff
@@ -583,6 +583,12 @@ def check_name_value(value, units=None, name=None, par_format=None):
 class Name(String):
     owl_uris = String.owl_uris + ("http://odahub.io/ontology#AstrophysicalObject",)
 
+class FileReference(String):
+    owl_uris = String.owl_uris + ("http://odahub.io/ontology#FileReference",)
```

Comment:
I think `String.owl_uris` is not needed here (and it changes nothing as far as I remember, because of the order the owl class hierarchy is matched with the python class hierarchy)

Reply:
ok, so can it be removed also for the `Name` class?